add on conflict clause in create table statement (#1860)
risingwavelabs/risingwave-docs: docs/transform/multiple-sink-into-table.md

---
id: multiple-sink-into-table

> slug: /multiple-sink-into-table

Reviewer: Don't forget to add it to `sidebar.js` as well

> title: multiple-sink-into-table

Reviewer: ```suggestion
title: Multiple Sink into table
```

Reviewer: Besides, regarding of terminology, how about call it "Table sink" among our docs? We usually say "sink into table" but this sounds like a verb word rather than noun.

---
<head>
  <link rel="canonical" href="https://docs.risingwave.com/docs/current/multiple-sink-into-table/" />
</head>

This guide tells a way to maintain a wide table whose columns are comes from different sources. Traditional data warehouse or ETL use a join query to do it. But the streaming join brings issues such as low efficiency and large memory consumption. 

> In some cases with limitation, use the [CREATE SINK INTO TABLE](/commands/sql-create-sink-into.md) and [ON CONFLICT clause](/commands/sql-create-table.md#pk-conflict-behavior) can save the resources and get high efficiency.

Reviewer: You need to add a new line for each paragraph according to Markdown spec. Lines without newlines will be rendered into one paragraph.

Ditto for many other occurances.

```suggestion

In some cases with limitation, use the [CREATE SINK INTO TABLE](/commands/sql-create-sink-into.md) and [ON CONFLICT clause](/commands/sql-create-table.md#pk-conflict-behavior) can save the resources and get high efficiency.
```

Now it's:

<img width="918" alt="image" src="https://github.com/risingwavelabs/risingwave-docs/assets/10192522/10a422d2-ae53-41d6-8887-792dbd1da6d4">

## Merge multiple sinks with the same primary key

:::note

>   Because the ON CONFLICT clause does not influence the Update or Delete operation, the sinks should be forced to append only, otherwise, the delete or update from any sink can delete the whole row.

Reviewer: Should not uppercase first character in "Update" "Delete" 

```suggestion
Keep in mind that the `ON CONFLICT` clause does not affect the update or delete events, the sinks should be forced to be append-only, otherwise, the delete or update events from any sink will delete the regarding row.
```
